### ISSUE TITLE
Increase dash version to 2.10 or greater

### ISSUE THREAD

**Issue by delsim** [GibbsConsulting/django-plotly-dash#491]
The dash version is currently limited to <2.10 as django-plotly-dash uses an internal API that is no longer present in 2.10
Refactoring of the code to avoid this (and any other steps) are needed to move to this version and beyond.

**Comment by Nizhurin** [comment]
It will be pretty cool if You can upgrade to dash > 2.0

**Comment by yzcheung** [comment]
The feature of exposing setProps to dash_clientside.clientSide_setProps was introduced in Dash version 2.16.1, which is quite a cool feature. It would be great if django-plotly-dash could keep up with this version update.

**Comment by mmc1718** [comment]
What would need doing to support more recent versions of Dash? Is #485 enough? I assume some code changes are required

**Comment by hairypalm** [comment]
Updating dash to the latest version (2.18.1) crashed my django app on _generate_meta_html().  However, I made this update to dash_wrapper.py and it started working.  Not sure if it passes all tests.  I think that _generate_meta_html() function in dash did something else compared to _generate_meta(), but it lets my app work.


```
dash_wrapper.py

. . .
    def index(self, *args, **kwargs):  # pylint: disable=unused-argument
        scripts = self._generate_scripts_html()
        css = self._generate_css_dist_html()
        config = self._generate_config_html()
        # metas = self._generate_meta_html()  # THIS DOESN'T EXIST
        metas = self._generate_meta() # THIS SEEMS TO WORK
        renderer = self._generate_renderer()
. . .

```

**Comment by GibbsConsulting** [comment]
This should now be addressed by V2.4.0 which has been released.
